subprojects/docs/src/samples/testKit/gradleRunner/junitQuickstart/kotlin/src/test/kotlin/org/gradle/sample/BuildLogicFunctionalTest.kt

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.sample
+
+// tag::functional-test-junit[]
```

Comment:
I would maybe move the tags in all samples below the imports. The import make the rendered page look unnecessary complicated in my mind. On the other hand, copy&pasting from the page might become a bit more difficult...